worker/uniter/modes.go

```diff
@@ -175,13 +194,10 @@ func ModeAbide(u *Uniter) (next Mode, err error) {
 		return nil, errors.Trace(err)
 	}
 	if !u.ranConfigChanged {
-		return continueAfter(u, newSimpleRunHookOp(hooks.ConfigChanged))
+		return continueAfter(u, "running config-changed hook", newSimpleRunHookOp(hooks.ConfigChanged))
 	}
 	if !opState.Started {
```

Comment:
This feels like it should be part of the op, but I think it's ok as a transitional state,